frontend/src/main/pages/SchoolCreatePage.js

```diff
@@ -0,0 +1,50 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import SchoolForm from "main/components/School/SchoolForm";
+import { Navigate } from "react-router-dom";
+import { useBackendMutation } from "main/utils/useBackend";
+import { toast } from "react-toastify";
+
+export default function SchoolCreatePage({ storybook = false }) {
+  const objectToAxiosParams = (school) => ({
+    url: "/api/schools/post",
+    method: "POST",
+    params: {
+      abbrev: school.abbrev,
+      name: school.name,
+      termRegex: encodeURIComponent(school.termRegex),
+      termDescription: school.termDescription,
+      termError: school.termError,
```

Comment:
I'm wondering if this parameter is still in the backend? 
And if it's required? And if the fact that it's been removed from the form is the reason for the error?

Reply:
I asked Aaron and he said you asked him to remove it, but only for the frontend, and already merged into main in some previous PRs (which means main is probably broken now).

Reply:
I have proposed a fix in #74